gradle.properties

```diff
@@ -29,4 +29,4 @@ AndroidAutoIdrive_GmapsApiKey=unset
 AndroidAutoIdrive_MapboxAccessToken=unset
 AndroidAutoIdrive_MapboxDownloadToken=unset
 AndroidAutoIdrive_SentryDsn=unset
-AndroidAutoIdrive_SpotifyApiKey=invalid
\ No newline at end of file
+AndroidAutoIdrive_SpotifyApiKey=unset
```

Comment:
[The build](https://github.com/BimmerGestalt/AAIdrive/blob/7cece396041ffbaeaae66b15f293bb847aff7f7a/app/build.gradle#L188) uses this special `invalid` to crash the build without the user specifically saying they want to disable the Spotify integration. If you do want this, you should set it to `unset` in your local gradle.properties, not in the repo file.

Reply:
reverted to original

Reply:
Thanks!